core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -202,6 +206,7 @@ public Boolean apply(Void input) {
   public void start() throws Exception {
     mJournalSystem.start();
     mJournalSystem.setMode(Mode.PRIMARY);
+    mSafeModeManager.enterSafeMode();
```

Comment:
the timer should start closer to where we start serving

Reply:
Done.